x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/transform/transforms_start_stop.yml

```diff
@@ -61,6 +61,15 @@ setup:
               }
             }
           }
+  - do:
+      cluster.put_settings:
+        body: >
+          {
+            "persistent": {
+              "logger.org.elasticsearch.xpack.transform.action": "DEBUG"
```

Comment:
I think you will also have to remove the debug in the teardown otherwise it will remain enabled for other tests too.